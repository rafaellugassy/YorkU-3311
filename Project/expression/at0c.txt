counting

start_of_set_enumeration
bool_value(true)
bool_value(false)
bool_value(true)
bool_value(false)
end_of_set_enumeration

analyze
-- count the number of true values (including duplicates)
simplify
