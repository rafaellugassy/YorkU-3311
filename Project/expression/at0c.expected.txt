  Expression currently specified: ?
  Report: Expression is initialized.
->counting
  Expression currently specified: (# ?)
  Report: OK.
->start_of_set_enumeration
  Expression currently specified: (# {?})
  Report: OK.
->bool_value(True)
  Expression currently specified: (# {True, ?})
  Report: OK.
->bool_value(False)
  Expression currently specified: (# {True, False, ?})
  Report: OK.
->bool_value(True)
  Expression currently specified: (# {True, False, True, ?})
  Report: OK.
->bool_value(False)
  Expression currently specified: (# {True, False, True, False, ?})
  Report: OK.
->end_of_set_enumeration
  Expression currently specified: (# {True, False, True, False})
  Report: OK.
->analyze
  Expression currently specified: (# {True, False, True, False})
  Report: (# {True, False, True, False}) is type-correct.
->simplify
  Expression currently specified: (# {True, False, True, False})
  Report: 2
